layer/lambda_utils/lambda_utils/header_config.py

```diff
@@ -41,10 +41,13 @@ class ClientRpDetailsHeader(AbstractHeader):
 
 
 class ConnectionMetadata(AbstractHeader):
-    pointer_types: list[str] = Field(alias="nrl.pointer-types")
+    pointer_types: list[str] = Field(alias="nrl.pointer-types", default_factory=list)
     ods_code: str = Field(alias="nrl.ods-code")
     ods_code_extension: str = Field(alias="nrl.ods-code-extension", default=None)
-    nrl_permissions: list[str] = Field(alias="nrl.permissions", default=[])
+    nrl_permissions: list[str] = Field(alias="nrl.permissions", default_factory=list)
+    enable_permissions_lookup: bool = Field(
```

Comment:
Im not sure the custom attribute can be a bool - will it be able to parse this?

Reply:
Seems to work fine:

![image](https://github.com/NHSDigital/NRLF/assets/19347498/1c0614b2-2fb2-4e6e-88ad-84229429b237)

Reply:
the value we pass to NRLF can certainly be a bool, more of a question about how we choose to parse this on the apigee side